wgpu-types/src/lib.rs

```diff
@@ -4503,58 +4503,42 @@ pub enum PollType<T> {
     ///
     /// On WebGPU, this has no effect. Callbacks are invoked from the
     /// window event loop.
-    WaitForSubmissionIndex(T),
-
-    /// Same as [`Self::WaitForSubmissionIndex`] but with a timeout.
-    WaitForSubmissionIndexWithTimeout {
+    Wait {
         /// Submission index to wait for.
-        submission_index: T,
+        ///
+        /// If not specified, will wait for the most recent submission at the time of the poll.
+        /// (by the time the method returns, more submissions may have taken place)
```

Comment:
style: In English, parentheses are for a fragmentary thought within a host sentence. If this is simply another sentence we want, let's remove the parentheses. Alternatively, we could put a paragraph between them and/or prepend N.B., viz., "… at the time of the poll. N.B. that by the time the method returns, …".